tests/features/00_smoke.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Smoke test
+  Simple test to check if test site is running.
+
+  Scenario: See front page
+    Given I am on "/"
+    Then I should see "Biblo"
```

Comment:
Yep. This feature was primarily meant to test that the Behat + Selenium setup is working.